packages/component-library/src/components/form/mt-link/mt-link.interactive.stories.ts

```diff
@@ -0,0 +1,65 @@
+import { within } from "@storybook/test";
+import { expect } from "@storybook/test";
+
+import meta, { type MtLinkMeta, type MtLinkStory } from "./mt-link.stories";
+
+export default {
+  ...meta,
+  title: "Interaction Tests/Form/mt-link",
+} as MtLinkMeta;
+
+export const VisualTestRenderLink: MtLinkStory = {
+  name: "Render link",
+  args: {
+    to: "/",
+  },
+};
+
+export const VisualTestRenderLinkPrimary: MtLinkStory = {
+  name: "Link primary",
+  args: {
+    variant: "primary",
+  },
+};
```

Comment:
This story is kind of unnecessary because it's the same as the one above.

Reply:
Thank you, I removed it.